src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -205,11 +205,31 @@ public override bool Execute ()
 			Log.LogDebugTaskItems ("  LibraryDirectories:", LibraryDirectories);
 			Log.LogDebugTaskItems ("  LibraryJars:", LibraryJars);
 
+			foreach (var issue in DisabledIssuesByVersion) {
+				if (lintToolVersion < issue.Value) {
+					DisabledIssues = CleanIssues (issue.Key, lintToolVersion, DisabledIssues, nameof (DisabledIssues));
+					EnabledIssues = CleanIssues (issue.Key, lintToolVersion, EnabledIssues, nameof (EnabledIssues) );
+				}
+			}
+
 			base.Execute ();
 
 			return !Log.HasLoggedErrors;
 		}
 
+		string CleanIssues (string issueToRemove, Version lintToolVersion, string issues, string issuePropertyName)
+		{
+			Regex issueReplaceRegex = new Regex ($"\b{issueToRemove}\b(,)?");
```

Comment:
You don't need the `,`. The `\b` word boundary will still be triggered at a comma.

```csharp
var r = new Regex(@"\bFoo\b");
r.Match ("Foo").Success;     // true
r.Match ("Foo,").Success;    // true
r.Match ("Foo,Bar").Success; // true
r.Match ("FooBar").Success;  // false
```
```